test/2_integration/005calculatedValueUpdate.test.js

```diff
@@ -77,17 +75,17 @@ describe('Calculated value test', () => {
     ).buildAndStore()
   })
 
-  after(async () => {
+  afterAll(async () => {
     if (survey) {
       await SurveyManager.deleteSurvey(Survey.getId(survey))
     }
   })
 
-  it('Calculated value updated', async () => {
+  test('Calculated value updated', async () => {
```

Comment:
jest also has it. what's the difference with test?

Reply:
It is just an alias... so no difference
https://jestjs.io/docs/en/api.html#testname-fn-timeout